### PR TITLE
License change to Apache License Version 2.0

### LINKED ISSUE #237: Petition to change license to Apache 2.0 (to match tesseract-ocr)
I want to use tesseract, and in turn pytesseract, but I don't want to include a GPLv3-licensed package before I know what I plant to do with my project.

Has any consideration been given to using a more permissive license such as MIT? Tesseract uses [Apache 2.0](https://github.com/tesseract-ocr/tesseract/blob/master/LICENSE), which is a fairly permissive license as well.

### PR DESCRIPTION
**Pull request by int3l**
Changing the license of this repository according to the vote from #237 (Petition to change license to Apache 2.0 (to match tesseract-ocr)).

Resolves #237

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -219,7 +219,7 @@ installed and in your PATH.
 
 LICENSE
 -------
-Python-tesseract is released under the GPL v3.
+Check the LICENSE file included in the Python-tesseract repository/distribution.
```
Comment (madmaze) [inline README.rst]: Can we update this to say that as of this version 0.3.1 it is Apache?

Comment (int3l) [reply README.rst]: Yep

Comment (int3l) [reply README.rst]: Done, republished the 0.3.1 release including the notice in the LICENSE section.

**Comment by madmaze** [comment]
I pushed out v0.3.1 to pypi last night

**Comment by int3l** [comment]
Thanks, I created PR conda-forge/pytesseract-feedstock#5 for the conda package as well.

**Pull request merged**